src/OpenTK.NT/Native/User32/Structs/RawInputDeviceList.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Runtime.InteropServices;
+
+using HANDLE = System.IntPtr;
+
+namespace OpenTK.NT.Native
+{
+    /// <summary>
+    /// Contains information about a raw input device.
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
```

Comment:
`struct` types in C# have the sequential layout type by default. Redundant, remove. There are multiple instances of this.